packages/i18n/src/locales/en_US.ts

```diff
@@ -167,6 +167,12 @@ const translations: Catalog = {
           }
         },
       },
+      AbstractCursor: {
```

Comment:
Why is this necessary here? I don't think we want to make this class visible to the user in any way or?

Can we rather exclude this one with some annotation on the `AbstractCursor` class, something like "@abstract" or "@nohelp"?

Reply:
And of course i've tagged random users ... 🤦

Reply:
> Can we rather exclude this one with some annotation on the AbstractCursor class, something like `@abstract` or `@nohelp`?

Yes, let me try that (I think @gribnoysup’s #784 also has the potential to make this kind of thing much easier in the future)

Reply:
@mcasimir Okay, done :) Feel free to take another look

Reply:
Yeah, totally can take inheritance chains while ignoring those abstract classes into account when generating metadata